styleGuide/util.css

```diff
@@ -0,0 +1,15 @@
+.mg-s {
+  margin: 1rem;
+}
+
+.mg-m {
+  margin: 2rem;
+}
+
+.pd-s {
+  padding: 1rem;
+}
+
+.pd-m {
+  padding: 2rem;
+}
```

Comment:
just a suggestion instead of using pd-m(padding medium/large/small) what you can do is you can use pd-1/2/3/4/5 this will you a bigger range. Keeping the base padding defined and using that every where inside var().